### PR TITLE
Docs release 1.9.0 csi 3956 nonfeature2

### PR DESCRIPTION
**Pull request by rivkap**

### CODE REVIEW

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -26,7 +26,7 @@
 * [Installing](content/installation/installation.md)
 	* [Compatibility and requirements](content/installation/install_compatibility_requirements.md)
 	* [Installing the operator and driver](content/installation/install_operator_driver.md)
```
Comment (oriyarde) [inline docs/SUMMARY.md:28]: what about this file?

File: docs/content/installation/install_driver_github.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 The operator for IBM® block storage CSI driver can be installed directly with GitHub. Installing the CSI (Container Storage Interface) driver is part of the operator installation process.
```
Comment (oriyarde) [inline docs/content/installation/install_driver_github.md:3]:
```suggestion
The IBM® block storage CSI driver operator can be installed directly with GitHub. Installing the CSI (Container Storage Interface) driver is part of the operator installation process.
```

File: docs/SUMMARY.md [hunk 3]
```diff
@@ -26,7 +26,7 @@
...
-		* [Installing the driver with OpenShift web console](content/installation/install_driver_openshift_web.md)
+		* [Installing the driver with the OpenShift web console](content/installation/install_driver_openshift_web.md)
 		* [Installing the driver with GitHub](content/installation/install_driver_github.md)
 		* [Installing the driver with OperatorHub.io](content/installation/install_driver_operatorhub.md)
```
Comment (oriyarde) [inline docs/SUMMARY.md:31]: what about this file?

File: docs/content/installation/install_driver_openshift_web.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ The Red Hat OpenShift Container Platform uses the following `SecurityContextCons
 
 3. Search for IBM block storage CSI driver.
```
Comment (oriyarde) [inline docs/content/installation/install_driver_openshift_web.md:19]: I can't comment on the third line of this file, but what about renaming there?

File: docs/SUMMARY.md (hunk 1 above)
Comment (rivkap) [reply docs/SUMMARY.md:28]: What is the question here? the names here are aligned with the content

File: docs/content/installation/install_driver_github.md (hunk 2 above)
Comment (rivkap) [reply docs/content/installation/install_driver_github.md:3]: I don't see a need to update this. i think that the way it is now is clearer

File: docs/content/installation/install_driver_openshift_web.md (hunk 4 above)
Comment (rivkap) [reply docs/content/installation/install_driver_openshift_web.md:19]: again, this isn't writing the actual _name_ of the operator but just referring to it. no need to update.
keeping as-is

> When using the Red Hat® OpenShift® Container Platform, the operator for IBM® block storage CSI driver can be installed directly from OpenShift web console, through the 

OperatorHub.

File: docs/SUMMARY.md (hunk 3 above)
Comment (oriyarde) [reply docs/SUMMARY.md:31]: yes, no, I'm referring to the new operator name, not to the "the"

File: docs/content/installation/uninstall_driver_openshift_web.md [hunk 5]
```diff
@@ -17,6 +17,6 @@ Perform the following steps in order to uninstall the CSI driver and operator th
 
     This deletes the CSI driver. Continue to step [5](#operator) to delete the operator for IBM block storage CSI driver.
 
-5. <a name="operator"></a>From the **Installed Operators** page, click on the **more** menu for the **Operator for IBM block storage CSI driver** and select **Uninstall Operator**.
```
[the remark this answers is not in the document]
Comment (oriyarde) [reply docs/content/installation/uninstall_driver_openshift_web.md:20]: I didn't mean to capitalize the O in operator. I meant using the new name

File: docs/SUMMARY.md (hunk 1 above)
Comment (oriyarde) [reply docs/SUMMARY.md:28]: I'm referring to [this](https://github.com/IBM/ibm-block-csi-driver/blob/v1.8.0/docs/content/installation/install_operator_driver.md) line:
>Install the operator for IBM® block storage CSI driver

File: docs/content/installation/uninstall_driver_openshift_web.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (rivkap) [reply docs/content/installation/uninstall_driver_openshift_web.md:20]: I still don't think that this needs to be updated. it's just describing what the operator is being used for. no need to update

File: docs/SUMMARY.md (hunk 3 above)
Comment (rivkap) [reply docs/SUMMARY.md:31]: I don't refer to the Operator by name here in the summary.MD file.

File: docs/content/release_notes/changelog_1.4.0.md [hunk 6]
```diff
@@ -4,7 +4,7 @@ IBM® block storage CSI driver 1.4.0 provided a range of enhancements and resolv
 
 -   New IBM Power Systems™ architecture support for Red Hat® OpenShift® 4.4 and 4.5
 -   Additional support for Kubernetes 1.19 and Red Hat OpenShift 4.4 and 4.5 with x86 architecture
--   Supports for IBM Z® for Red Hat OpenShift 4.5
+-   Adds support for IBM Z® for Red Hat OpenShift 4.5
```
[the remark this answers is not in the document]
Comment (rivkap) [reply docs/content/release_notes/changelog_1.4.0.md:7]: done

File: docs/SUMMARY.md (hunk 3 above)
Comment (oriyarde) [reply docs/SUMMARY.md:31]: i'm referring to the `install_driver_operatorhub.md` file content (I can't comment on it in this diff)

File: docs/content/release_notes/changelog_1.4.0.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (oriyarde) [reply docs/content/release_notes/changelog_1.4.0.md:7]: I don't see any change here

[the remark this answers is not in the document]
Comment (rivkap) [reply docs/content/release_notes/changelog_1.4.0.md]: forgot to commit :) done

**Pull request merged**